### PR TITLE
The throw state show when press the key W+S

### PR DESCRIPTION
**Pull request by onon1101**
this pr can show the gui state 
image

Additionally, creating special effects for the thrown dagger can be challenging, as it requires knowledge of SDL2. Therefore, I will not write that part.

### CODE REVIEW

**Review by HeavenManySugar — APPROVED** [review]
LGTM

File: include/Player/Equipment/Throw.h [hunk 1]
```diff
@@ -35,13 +37,23 @@ class Throw final : public IEquipment {
...
     std::shared_ptr<GameElement> m_Throw = std::make_shared<GameElement>();
+
+    std::shared_ptr<Util::Image> m_NormalSlot;
+    std::shared_ptr<Util::Image> m_ThrowSlot;
+
+private:
+    void GenWin();
+    void GenItem();
+    void GeneralUpperText();
+    void GeneralLowerText();
```
Comment (HeavenManySugar) [inline include/Player/Equipment/Throw.h:56]: WTF
這是大寫和小寫的意思嗎

Comment (onon1101) [reply include/Player/Equipment/Throw.h:56]: This implies that the display consists of two lines: the first line and the second line.

**Comment by onon1101** [comment]
Please review again.

**Pull request merged**